### ISSUE TITLE
Accessibility: List of Email and Social Media icons missing link text

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#1687]
The 5 list item: Email, Facebook, Twitter, Google Plus, and LinkedIn Share links are missing TEXT which screen reader users rely on to know what the links lead to. See correctly coded example from web.mit.edu here:

```<a href="http://www.facebook.com/MITnews" class="social facebook" title="Connect with us on Facebook" target="_blank">MIT on Facebook</a>```

In the example above, the screen reader reads: "Link, MIT on Facebook"

**Comment by aliceriot** [comment]
It looks like the template we use for the sharing buttons we do something like:

```html
<li class="rrssb-email">
  <a href="">
    <span class="rrssb-icon"></span>
    <span class="rrssb-text">email</span>
  </a>
</li>
```

if I get a reference to the `<a>` tag and call `.textContent` on it I get 'email'. What exactly do we need to change?

**Comment by pdpinch** [comment]
Sounds like we've already fixed this? I'll check with the accessibility folks. 

See https://github.mit.edu/ODLaccessibility/MicroMasters-Portal/issues/4

**Comment by aliceriot** [comment]
thanks, I'll take a look at that
